src/elements/common/base.scss

```diff
@@ -1,19 +1,19 @@
 @import './variables';
-@import '~box-react-ui/lib/styles/common/buttons';
-@import '~box-react-ui/lib/styles/common/inline-notifications';
-@import '~box-react-ui/lib/styles/modifiers/accessibility';
-@import '~box-react-ui/lib/styles/common/overlay';
-@import '~box-react-ui/lib/styles/common/pills';
-@import '~box-react-ui/lib/components/pill-selector-dropdown/PillSelector';
-@import '~box-react-ui/lib/components/notification/Notification';
+@import '../../styles/common/buttons';
+@import '../../styles/common/inline-notifications';
+@import '../../styles/modifiers/accessibility';
```

Comment:
I'm not sure if there's a convention today, but having a common import order would be helpful. Something like:

1. variables
1. common rules/helpers
1. components
1. modifiers
1. overrides

Reply:
Can do this later